osh/hub/scan/models.py

```diff
@@ -1283,3 +1283,23 @@ def csmock_args(self):
             # there are no arguments, this is not an error
             logger.info("No csmock arguments for profile '%s'", self)
             return ''
+
+
+class RetentionPolicySettingsManager(models.Manager):
+    def get_parameter(self, category):
+        try:
+            days = self.get(category=category)
+        except ObjectDoesNotExist:
+            logger.error("category %s does not exist", category)
+            raise ObjectDoesNotExist("category %s does not exist" % category)
+        return days
+
+
+class RetentionPolicySetting(models.Model):
+    category = models.CharField(max_length=128, blank=False, null=False)
+    days = models.PositiveIntegerField(
+        default=14,
+        help_text="Number of days before the text is deleted "
+    )
+
+    objects = RetentionPolicySettingsManager()
```

Comment:
Right now, there are only two elements in the corresponding fixture.  If we definitely plan to add other settings in the future, having a separate model is a good idea.  If we don't, I suggest to use the `AppSettings` model instead which already contains, e.g. the number of days after which a not waived scan is overdue.

Reply:
For this, I had a discussion with @kdudka and we decided to create a new model for this.That's why we didn't use the AppSetttings model.

Reply:
Cool!  Thank you for the explanation!

Reply:
Yes, we decided to use a separate table because we are going to append additional rows in the future.  Failed scans and user-initiated scans are the low-hanging fruit.  But for scans initiated using a service principal, we need to distinguish scans in CI vs. scans of production builds.  For production builds we need to distinguish the ones that were in the end shipped to customers vs. the builds that were overridden by a subsequent build, etc.